### ISSUE TITLE
Numpy encoder deprecated ?

### ISSUE THREAD

**Issue by neutrinoceros** [uiri/toml#274]
Hi, I am trying to use Toml (10.0) with numpy but the example given in the readme doesn't seem to work anymore

```python
import numpy as np
import toml
a = np.arange(0, 10, dtype=np.double) 
output = {'a': a} 
toml.dumps(output, encoder=toml.TomlNumpyEncoder())
```
output
```python
AttributeError: module 'toml' has no attribute 'TomlNumpyEncoder'
```

is this a wanted change or a bug ? in any case at least the documentation should be updated.
Thanks !

**Comment by neutrinoceros** [comment]
ah, turns out it's actually a new feature coming from 10.1 maybe it's worth noting that in the documentation since 10.0 is still the most recent version on pypi to this day

**Comment by mileslucas** [comment]
Fixed by #248. I've been waiting for a PyPI release for ~7 months now along with others.

**Comment by neutrinoceros** [comment]
This is indeed solved as for version 0.10.2, thanks !
